packages/manager/src/features/Linodes/LinodesCreate/SelectLinodePanel.tsx

```diff
@@ -72,9 +72,11 @@ const SelectLinodePanel = (props: Props) => {
           <>
             <StyledPaper data-qa-select-linode-panel>
               {error && <Notice text={error} variant="error" />}
-              {notice && !disabled && (
-                <Notice text={notice.text} variant={notice.level} />
-              )}
+              {notices &&
+                !disabled &&
+                notices.map((notice, i) => (
+                  <Notice key={i} text={notice.text} variant={notice.level} />
+                ))}
```

Comment:
Optional: I think it would look better if the spacing was tightened up

```suggestion
              <Stack gap={0.5} mb={2}>
                {error && (
                  <Notice
                    spacingBottom={0}
                    spacingTop={0}
                    text={error}
                    variant="error"
                  />
                )}
                {notices &&
                  !disabled &&
                  notices.map((notice, i) => (
                    <Notice
                      key={i}
                      spacingBottom={0}
                      spacingTop={0}
                      text={notice.text}
                      variant={notice.level}
                    />
                  ))}
              </Stack>
```

Reply:
![Screenshot 2024-01-18 at 11 48 10 AM](https://github.com/linode/manager/assets/115251059/c3b4cc16-a617-4d3c-8ca6-d3ca951636f9)